### ISSUE TITLE
Portuguese (pt-pt) translation (0.6.10)

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#262]
```
Translated strings.xml to portuguese (pt-pt).

Only raw xml, no patch.




```

Original issue reported on code.google.com by `jose.rui...@gmail.com` on 26 Oct 2011 at 4:18

Attachments:
- [values_pt.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-262/comment-0/values_pt.zip)

**Comment by GoogleCodeExporter** [comment]
Original comment by `sdeva...@gmail.com` on 26 Oct 2011 at 6:38
- Added labels: **Type-Translation**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
```
Thanks, looks good. Committed in r196.

Nice work! Also added you to the "Authors"-list.
```

Original comment by `oskar.wi...@gmail.com` on 30 Oct 2011 at 2:00
- Changed state: **Ready**
- Added labels: **Milestone-0.6.11**

**Comment by GoogleCodeExporter** [comment]
```
Added content_actorconditions.xml
```

Original comment by `jose.rui...@gmail.com` on 1 Nov 2011 at 2:49

Attachments:
- [values_pt.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-262/comment-3/values_pt.zip)

**Comment by GoogleCodeExporter** [comment]
```
Thanks, committed in r200.
```

Original comment by `oskar.wi...@gmail.com` on 2 Nov 2011 at 8:55

**Comment by GoogleCodeExporter** [comment]
```
Added content_monsterlist.xml
```

Original comment by `jose.rui...@gmail.com` on 3 Nov 2011 at 3:24

Attachments:
- [values_pt.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-262/comment-5/values_pt.zip)

**Comment by GoogleCodeExporter** [comment]
```
Added content_itemlist.xml and did some correction in content_monsterlist.xml.
```

Original comment by `jose.rui...@gmail.com` on 6 Nov 2011 at 2:57

Attachments:
- [values_pt.7z](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-262/comment-6/values_pt.7z)

**Comment by GoogleCodeExporter** [comment]
```
Corrected values in content_actorconditions.xml and strings.xml.
Added 0.6.11 texts in content_itemlist.xml, content_actorconditions.xml and 
content_monsterlist.xml
```

Original comment by `jose.rui...@gmail.com` on 25 Nov 2011 at 5:21

Attachments:
- [values_pt.zip](https://storage.googleapis.com/google-code-attachments/andors-trail/issue-262/comment-7/values_pt.zip)

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.11. Closing issue.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jun 2012 at 9:31
- Changed state: **Fixed**
